### PR TITLE
fix(core): Fix migrations for MySQL/MariaDB

### PR DESCRIPTION
**Pull request by netroy**
fixes 
https://community.n8n.io/t/n8n-doesnt-start-after-updating-to-0-234-0/27751
https://community.n8n.io/t/last-session-crashed-after-migration/27758

### CODE REVIEW

File: packages/cli/src/databases/migrations/mysqldb/1690000000030-SeparateExecutionData.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ export class SeparateExecutionData1690000000030 implements ReversibleMigration {
 			`CREATE TABLE ${tablePrefix}execution_data (
 				executionId int(11) NOT NULL primary key,
 				workflowData json NOT NULL,
-				data TEXT NOT NULL,
```
Comment (netroy) [inline packages/cli/src/databases/migrations/mysqldb/1690000000030-SeparateExecutionData.ts:9]: This fixes the issue for anyone who hasn't upgraded yet.

File: packages/cli/src/databases/migrations/mysqldb/1690000000031-FixExecutionDataType.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import type { MigrationContext, IrreversibleMigration } from '@db/types';
+
+export class FixExecutionDataType1690000000030 implements IrreversibleMigration {
+	async up({ queryRunner, tablePrefix }: MigrationContext) {
+		await queryRunner.query(
+			'ALTER TABLE `' + tablePrefix + 'execution_data` MODIFY COLUMN `data` MEDIUMTEXT',
```
Comment (netroy) [inline packages/cli/src/databases/migrations/mysqldb/1690000000031-FixExecutionDataType.ts:6]: This ensures that schema is consistent for anyone who managed to upgrade to `0.234.x` or `0.235.x`.

**Review by flipswitchingmonkey — DISMISSED** [review]

File: packages/cli/src/databases/migrations/mysqldb/1690000000031-FixExecutionDataType.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import type { MigrationContext, IrreversibleMigration } from '@db/types';
+
+export class FixExecutionDataType1690000000030 implements IrreversibleMigration {
```
Comment (ivov) [inline packages/cli/src/databases/migrations/mysqldb/1690000000031-FixExecutionDataType.ts:3]: Class name timestamp doesn't match filename timestamp.

File: packages/cli/src/databases/migrations/mysqldb/1690000000031-FixExecutionDataType.ts (hunk 2 above)
Comment (ivov) [reply packages/cli/src/databases/migrations/mysqldb/1690000000031-FixExecutionDataType.ts:6]: Can we add the reason as an actual comment? More generally, maybe we should consider starting to add short text summaries to migrations.

**Review by ivov — CHANGES_REQUESTED** [review]

File: packages/cli/src/databases/migrations/mysqldb/1690000000031-FixExecutionDataType.ts (hunk 2 above)
Comment (netroy) [reply packages/cli/src/databases/migrations/mysqldb/1690000000031-FixExecutionDataType.ts:14]: done.

**Review by ivov — APPROVED** [review]

**Pull request merged**